compose/backend.yml

```diff
@@ -139,6 +139,10 @@ services:
       - 5555:5555
     networks:
       - ccd-network
+    #provide custom Idam roles when using Idam Stub.
+    #uncomment and add the desired roles to idam_stub_get_details_custom.json. Then restart the Stub Service
+#    volumes:
+#      - ${PWD}/resources/idam_stub_get_details_custom.json:/opt/app/wiremock/mappings/idam_get_details.json
```

Comment:
Can't we register wire-mock mappings dynamically with curl?

Reply:
@smathangi good point:
1) This is more a permanent solution. Dynamic registration is lost when the service is stopped
2) dynamic registration is more complex. I think this approach is simpler, especially for non devs

Reply:
👍